arex-web-model-contract/src/main/java/com/arextest/web/model/contract/contracts/config/application/ApplicationOperationConfiguration.java

```diff
@@ -21,4 +21,5 @@ public class ApplicationOperationConfiguration extends AbstractConfiguration imp
     private Set<String> operationTypes;
     private String operationResponse;
     private Integer recordedCaseCount;
+    private List<Dependency> dependencies;
```

Comment:
Considering the size of the contract, whether a separate interface outputs the contract